internal/cephfs/core/clone.go

```diff
@@ -28,16 +28,17 @@ import (
 
 // cephFSCloneState describes the status of the clone.
 type cephFSCloneState struct {
-	state    admin.CloneState
-	errno    string
-	errorMsg string
+	state          admin.CloneState
+	progressReport admin.CloneProgressReport
+	errno          string
+	errorMsg       string
 }
 
 // CephFSCloneError indicates that fetching the clone state returned an error.
-var CephFSCloneError = cephFSCloneState{}
+var CephFSCloneError = &cephFSCloneState{}
```

Comment:
Why is the change from value to reference required ?

Reply:
Because adding a new struct member to the `cephFSCloneState` was making the size of the struct too big, the CI was complaining for the same [here](https://github.com/ceph/ceph-csi/actions/runs/11548533844/job/32140093334). That's the reason, we need to have this change.

Reply:
Understood, thanks!